### PR TITLE
Update ExprVelocity.java

### PR DESCRIPTION
**Pull request by TheLimeGlass**
Clean up of the velocity/vector expression and possible NullPointerException fix.

Target Minecraft versions: *Any*

### CODE REVIEW

File: src/main/java/ch/njol/skript/expressions/ExprVelocity.java [hunk 1]
```diff
@@ -62,38 +63,38 @@ protected String getPropertyName() {
 	@Override
 	@SuppressWarnings("null")
 	public Class<?>[] acceptChange(final Changer.ChangeMode mode) {
-		if ((mode == Changer.ChangeMode.SET || mode == Changer.ChangeMode.ADD || mode == Changer.ChangeMode.REMOVE || mode == Changer.ChangeMode.DELETE))
+		if (!(mode == Changer.ChangeMode.REMOVE_ALL))
```
Comment (xXAndrew28Xx) [inline src/main/java/ch/njol/skript/expressions/ExprVelocity.java]: Why not use a != operator instead?

File: src/main/java/ch/njol/skript/expressions/ExprVelocity.java [hunk 2]
```diff
@@ -62,38 +63,38 @@ protected String getPropertyName() {
...
 	@Override
 	@SuppressWarnings("null")
-	public void change(final Event e, final @Nullable Object[] delta, final Changer.ChangeMode mode) throws UnsupportedOperationException {
-		for (final Entity ent : getExpr().getArray(e)) {
-			if (ent == null)
-				return;
+	public void change(final Event event, final @Nullable Object[] delta, final Changer.ChangeMode mode) throws UnsupportedOperationException {
+		if (delta == null) return;
+		for (final Entity entity : getExpr().getArray(event)) {
+			if (entity == null) return;
```
Comment (xXAndrew28Xx) [inline src/main/java/ch/njol/skript/expressions/ExprVelocity.java]: Wouldn't it make more sense to `continue;` (skipping over null elements) instead?

**Comment by bensku** [comment]
I see you created this in Github UI. Are you aware of that I require pull requesters to test their code?

File: src/main/java/ch/njol/skript/expressions/ExprVelocity.java (hunk 2 above)
Comment (TheLimeGlass) [reply src/main/java/ch/njol/skript/expressions/ExprVelocity.java]: That's the way Sashie had it, I would add a null check for the expression with the delta null check.

**Comment by Snow-Pyon** [comment]
Are you gonna make requested changes? Closing otherwise.

**Comment by Snow-Pyon** [comment]
Closing as no response was given.

File: src/main/java/ch/njol/skript/expressions/ExprVelocity.java (hunk 2 above)
Comment (Sashie) [reply src/main/java/ch/njol/skript/expressions/ExprVelocity.java]: don't throw me under the bus, it was bi0s code to begin with...

**Pull request closed without merging**